server/services/interventionsService.test.ts

```diff
@@ -279,6 +279,189 @@ pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, pr
         '5352cfb6-c9ee-468c-b539-434a3e9b506e',
       ])
     })
+
+    it('returns the updated referral when setting additionalNeedsInformation and accessibilityNeeds', async () => {
+      await provider.addInteraction({
+        state: 'a draft referral with ID dfb64747-f658-40e0-a827-87b4b0bdcfed exists',
+        uponReceiving: 'a PATCH request to set additionalNeedsInformation and accessibilityNeeds',
+        withRequest: {
+          method: 'PATCH',
+          path: '/draft-referral/dfb64747-f658-40e0-a827-87b4b0bdcfed',
+          headers: {
+            Accept: 'application/json',
+            'Content-Type': 'application/json',
```

Comment:
total nit pick - but why is this field in quotes but the others are not?

Reply:
I reckon it's an ESLint or Prettier thing - only quotes them if they need to be quoted (keys with special characters need to be quoted in JS I believe)